seed/views/v3/organizations.py

```diff
@@ -129,3 +339,999 @@ def column_mappings(self, request, pk=None):
+            cycle_id__in=cycles
+        )
+        organization = Organization.objects.get(pk=organization_id)
+        results = []
+        for cycle in cycles:
+            property_views = all_property_views.filter(cycle_id=cycle)
+            count_total = []
+            count_with_data = []
+            data = []
+            for property_view in property_views:
+                property_pk = property_view.property_id
+                if property_view.property.campus and campus_only:
+                    count_total.append(property_pk)
+                    result = self.get_data(property_view, x_var, y_var)
+                    if result:
+                        result['yr_e'] = cycle.end.strftime('%Y')
+                        data.append(result)
+                        count_with_data.append(property_pk)
+                elif not property_view.property.campus:
+                    count_total.append(property_pk)
+                    result = self.get_data(property_view, x_var, y_var)
+                    if result:
+                        result['yr_e'] = cycle.end.strftime('%Y')
+                        de_unitted_result = apply_display_unit_preferences(organization, result)
+                        data.append(de_unitted_result)
+                        count_with_data.append(property_pk)
+            result = {
+                "cycle_id": cycle.pk,
+                "chart_data": data,
+                "property_counts": {
+                    "yr_e": cycle.end.strftime('%Y'),
+                    "num_properties": len(count_total),
+                    "num_properties_w-data": len(count_with_data),
+                },
+            }
+            results.append(result)
+        return results
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_string_field(
+                'x_var',
+                required=True,
+                description='Raw column name for x axis'
+            ),
+            AutoSchemaHelper.query_string_field(
+                'y_var',
+                required=True,
+                description='Raw column name for y axis'
+            ),
+            AutoSchemaHelper.query_string_field(
+                'start',
+                required=True,
+                description='Start time, in the format "2018-12-31T23:53:00-08:00"'
+            ),
+            AutoSchemaHelper.query_string_field(
+                'end',
+                required=True,
+                description='End time, in the format "2018-12-31T23:53:00-08:00"'
+            ),
+            AutoSchemaHelper.query_string_field(
+                'campus_only',
+                required=False,
+                description='If true, includes campuses'
+            ),
+        ]
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_member')
+    @action(detail=True, methods=['GET'])
+    def report(self, request, pk=None):
```

Comment:
Maybe add a brief description here on what this endpoint is doing?

Reply:
@adrian-lara can you double check my description for this and report_aggregated?

Reply:
The report-related descriptions look good to me! 💯